app/_src/kic-v2/guides/prometheus-grafana.md

```diff
@@ -2,115 +2,81 @@
-$ helm install mykong kong/kong --namespace kong --set serviceMonitor.enabled=true --set serviceMonitor.labels.release=promstack
-```
-
-By default, kube-prometheus-stack [selects ServiceMonitors and PodMonitors by a
-`release` label equal to the release name](https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-19.0.1/charts/kube-prometheus-stack/values.yaml#L2128-L2169). The `labels` setting here adds a label matching the
-`promstack` release name from the example.
-
-### Enable Prometheus plugin in Kong
-
-We will enable the Prometheus plugin in Kong at the global level, meaning
-each request that flows into the Kubernetes cluster gets tracked in Prometheus:
+    ```yaml
+    prometheus:
+      prometheusSpec:
+        scrapeInterval: 10s
+        evaluationInterval: 30s
+    grafana:
+      persistence:
+        enabled: true  # enable persistence using Persistent Volumes
+      dashboardProviders:
+        dashboardproviders.yaml:
+          apiVersion: 1
+          providers:
+          - name: 'default' # Configure a dashboard provider file to
+            orgId: 1        # put Kong dashboard into.
+            folder: ''
+            type: file
+            disableDeletion: false
+            editable: true
+            options:
+              path: /var/lib/grafana/dashboards/default
+      dashboards:
+        default:
+          kong-dash:
+            gnetId: 7424  # Install the following Grafana dashboard in the
+            revision: 11  # instance: https://grafana.com/dashboards/7424
+            datasource: Prometheus
+          kic-dash:
+            gnetId: 15662
+            datasource: Prometheus
+    
+    ```
+
+1. To install Prometheus and Grafana, execute the following, specifying the path to the `values-monitoring.yaml` file that you created:
+
+    ```bash
+    $ kubectl create namespace monitoring
+    $ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    $ helm install promstack prometheus-community/kube-prometheus-stack --namespace monitoring --version 52.1.0 -f values-monitoring.yaml
+    ```
+
+## Enable ServiceMonitor
+
+1. To enable ServiceMonitor, set `gateway.serviceMonitor.enabled=true`.
+
+    ```bash
+    $ helm upgrade kong kong/ingress -n kong --set gateway.serviceMonitor.enabled=true --set gateway.serviceMonitor.labels.release=promstack
+    ```
+
+    By default, kube-prometheus-stack [selects ServiceMonitors and PodMonitors by a`release` label equal to the release name](https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-19.0.1/charts/kube-prometheus-stack/values.yaml#L2128-L2169). The `labels` setting here adds a label matching the `promstack` release name from the example.
+
+1. Enable the Prometheus plugin in Kong at the global level, so that each request that flows into the Kubernetes cluster gets tracked in Prometheus:
 
 ```bash
 $ echo 'apiVersion: configuration.konghq.com/v1
```

Comment:
I am not able to fix the formatting on this because when I indent this by 4 spaces and then copy the output in the terminal i get yaml liniting errors.